src/inject/dynamic-theme/index.ts

```diff
@@ -54,6 +57,19 @@ function createOrUpdateScript(className: string, root: ParentNode = document.hea
     return element;
 }
 
+/**
+ * Note: This function is used only with MV3.
+ * String passed as src parameter must be included in web_accessible_resources manifest key.
+ */
+function createScriptMV3(src: string, args: any = undefined) {
+    const element = document.createElement('script');
+    element.src = chrome.runtime.getURL(src);
+    if (args !== undefined) {
+        element.setAttribute('args', JSON.stringify(args));
```

Comment:
The `args` attribute seems to be non-standard. Maybe let's use `element.dataset.args = ...`? (It will create `dataset-args` attribute)